### PR TITLE
Additional documenation

### PR DESCRIPTION
**Pull request by geolich**
Add info about deprecation of authentication endpoint
Add new webhook model
Add Status Page page

### CODE REVIEW

File: source/includes/_statuspage.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Status Page
+Our status page delivers the realtime status about the functionality of our services.
```
Comment (laurin-no) [inline source/includes/_statuspage.md]: should be "real time status of"

File: source/includes/_statuspage.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+Incidents or Maintenance updates will also be posted there with timely updates about the status and reason of the incident/maintenance.
```
Comment (laurin-no) [inline source/includes/_statuspage.md]: should be "Incident or maintenance"

**Review by laurin-no — CHANGES_REQUESTED** [review]
reviewed, left some comments

File: source/includes/_userprovisioning.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Please note that the base URL for user provisioning is
 
-`https://services.1brd.com`
+`https://profile.services.1brd.com`
```
Comment (fgrutsch) [inline source/includes/_userprovisioning.md:5]: shouldnt it be customer.services.1brd.com? profile was the old name of the service

Comment (geolich) [reply source/includes/_userprovisioning.md:5]: It is profile still as customers already use this API and we need to tell them about the change

**Review by laurin-no — APPROVED** [review]

**Pull request merged**